_episodes/03-create.md

```diff
@@ -70,7 +70,7 @@ $ git status
 ~~~
 {: .bash}
 
-If you are using a different version of git than I am, then then the exact
+If you are using a different version of git, the exact
```

Comment:
Sounds good, but know you are it: could you please put quotes around git in this line? So git -> `git`. We use "Git" (capitalized) when talking about the protocol, a Git repository, or the system itself. We use `git` when talking about the implementation as a software program.

Reply:
Will do!